lib/cors-anywhere.js

```diff
@@ -50,8 +50,11 @@ function isValidHostName(hostname) {
  * @param headers {object} Response headers
  * @param request {ServerRequest}
  */
-function withCORS(headers, request) {
+function withCORS(headers, request, maxAge) {
   headers['access-control-allow-origin'] = '*';
```

Comment:
Please use the following instead of passing an extra parameter to `withCORS`:
`var maxAge = req.corsAnywhereRequestState.location;`